assests/javascript/index.js

```diff
@@ -0,0 +1,73 @@
+const baseUrl = "http://127.0.0.1:3030";
+
+const newTask = document.getElementById("task-input");
+const addTaskIcon = document.getElementById("add-task-icon");
+const taskBox = document.getElementById("task-box");
+const newTaskErrorMassage = document.getElementById("title-error");
+const networkErrorMassage = document.getElementById("network-error");
+
+newTask.addEventListener(
+  "keydown",
+  (event) => {
+    if (event.key === "Enter") {
+      handleNewTask(newTask.value);
+      newTask.value = "";
+    }
+  },
+  false
+);
+
+addTaskIcon.addEventListener("click", () => {
+  setNewTask(newTask.value);
+  newTask.value = "";
```

Comment:
Move this to a function call it `clearTaskInput` and re-use it.

Reply:
In the case of error we should not clear the input. so this can move in the add task function.

Reply:
1.How can I use `input.value` in external function?
2.You are right the input in already empty. Fixed! <3